### PR TITLE
[8.0] Making elasticjobparameters monitorable

### PR DESCRIPTION
**Pull request by rupozzi**
In order to make the elasticjobparameters readable by Grafana a timestamp needs to be added. 
Also, making the values of the parameters into string values will enable better monitoring options.

BEGINRELEASENOTES

*WMS
CHANGE: Addded timestamp to elasticjobparameters and changed parameter values into strings

ENDRELEASENOTES

### CODE REVIEW

File: src/DIRAC/WorkloadManagementSystem/DB/ElasticJobParametersDB.py [hunk 1]
```diff
@@ -28,12 +27,18 @@
       - setJobParameter()
       - deleteJobParameters()
 """
-import hashlib
-
+from xmlrpc.client import boolean
```
Comment (fstagni) [inline src/DIRAC/WorkloadManagementSystem/DB/ElasticJobParametersDB.py:30]: ?

File: src/DIRAC/WorkloadManagementSystem/DB/ElasticJobParametersDB.py [hunk 2]
```diff
@@ -64,116 +70,193 @@ def __init__(self, parentLogger=None):
...
 
         :param self: self reference
-        :param int jobID: Job ID
-        :param list parameters: list of tuples (name, value) pairs
-
+        :param jobID: Job ID
+        :param parameters: list of tuples (name, value) pairs
         :returns: S_OK/S_ERROR as result of indexing
         """
+        print("USING THIS METHOD AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA")
```
Comment (fstagni) [inline src/DIRAC/WorkloadManagementSystem/DB/ElasticJobParametersDB.py:156]: Yeah! :)

File: src/DIRAC/WorkloadManagementSystem/DB/ElasticJobParametersDB.py (hunk 1 above)
Comment (rupozzi) [reply src/DIRAC/WorkloadManagementSystem/DB/ElasticJobParametersDB.py:30]: Sometimes VScode automatically adds random imports from what I write :/

Comment (andresailer) [reply src/DIRAC/WorkloadManagementSystem/DB/ElasticJobParametersDB.py:30]: `git add --patch` is a beautiful way to see these things and not commit them.

File: src/DIRAC/WorkloadManagementSystem/DB/ElasticJobParametersDB.py [hunk 3]
```diff
@@ -193,16 +272,30 @@ def deleteJobParameters(self, jobID, paramList=None):
...
-
         for param in paramList:
             paramFilter = self._Q("term", Name=param)
             combinedFilter = jobFilter & paramFilter
 
             s = self.dslSearch.query("bool", filter=combinedFilter)
             s.delete()
 
+    def _deleteInNewIndex(self, jobID: int, paramList=None) -> dict:
+        """Deletes a document with this jobID in the old index"""
```
Comment (fstagni) [inline src/DIRAC/WorkloadManagementSystem/DB/ElasticJobParametersDB.py:283]: new or old?

**Comment by fstagni** [comment]
This looks OK. 

But, it's "temporary", as we don't want to leave "forever" with 2 index names and especially 2 different mappings. So, as soon as this PR will be merged, please do create another PR (that will be targeting still the integration branch, but that will be merged for DIRAC version 8.1), where the old index is not even looked up anymore.

**Comment by fstagni** [comment]
Fix the conflict then we can merge this one

**Review by fstagni — APPROVED** [review]

**Pull request merged**